### PR TITLE
Feature/con 709 index pattern

### PR DESCRIPTION
**Pull request by ChristopherGillis**

### CODE REVIEW

File: assets/scss/cruk-base.scss [hunk 1]
```diff
@@ -65,21 +65,22 @@
  */
 @import "base/type";
 
-@import "partials/animations";
```
Comment (heylookalive) [inline assets/scss/cruk-base.scss]: Animations was first as others reference it, like alerts - did this not cause a compilation error?

File: assets/js/cruk-base/plugin-alphabetic-list.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            function(i, li) {
+                var listItemText = $(li).text().toUpperCase(), searchText = that.value.toUpperCase();
+                return ~listItemText.indexOf(searchText);
+            }
+        );
+
+        $allListElements.hide();
+        $matchingListElements.show();
+    });
+});
```
Comment (heylookalive) [inline assets/js/cruk-base/plugin-alphabetic-list.js]: Please correct indentation to spaces, 2 per tab

Comment (heylookalive) [inline assets/js/cruk-base/plugin-alphabetic-list.js]: Could we add a data attribute to the input half of this pattern to target the list part by ID please? We then avoid any issues where multiple instances of things are on a single page. We wouldn't expect this to happen but I've heard crazier things.

File: assets/scss/cruk-base.scss (hunk 1 above)
Comment (ChristopherGillis) [inline assets/scss/cruk-base.scss]: Nah, no errors... Had a look and everything looks fine... The fact that we are extending class selectors (which isn't ideal anyway) means that import order doesn't matter.

**Comment by ChristopherGillis** [comment]
Cool, I've fixed that now. If an ID is provided in `data-cr-filter-list` it will target that, otherwise it will target every instance of `.cr-index` on the page.

**Pull request merged**